build/vsts-ci.yml

```diff
@@ -257,6 +256,14 @@ phases:
       msbuildVersion: 15.0
     continueOnError: true
 
+  - task: NuGetCommand@2 
```

Comment:
[As I mentioned here](https://github.com/dotnet/machinelearning/pull/4406/files#r339876679), I don't think it is worth keeping this.